far/FarEng.hlf.m4

```diff
@@ -2799,95 +2799,160 @@ saved path.
-    #Ctrl+Shift+Right#   End of lines on the screen
-    #Home, Ctrl+Home#    Start of file
-    #End, Ctrl+End#      End of file
-
-    #F1#                 Help
-    #F2#                 Toggle line wrap/unwrap
-    #Shift+F2#           Toggle wrap type (letters/words)
-    #F4#                 Toggle text/hex or dump/hex mode
-    #Shift+F4#           Select viewer mode (text/hex/dump)
-    #F6#                 Switch to ~editor~@Editor@
-    #F7#                 ~Search~@ViewerSearch@
-    #Shift+F7, Space#    Continue search
-    #Alt+F7#             Continue search in "reverse" mode
-    #F8#                 Toggle OEM/ANSI code page
-    #Shift+F8#           Select code page
-    #Alt+F8#             ~Change current position~@ViewerGotoPos@
-    #Alt+F9#             Toggles the size of the Far console window
-    #Alt+Shift+F9#       Call ~Viewer settings~@ViewerSettings@ dialog
-    #Numpad5,F3,F10,Esc# Quit
-    #Ctrl+F10#           Position to the current file.
-    #F11#                Call "~Plugin commands~@Plugins@" menu
-    #Alt+F11#            Display ~view history~@HistoryViews@
-    #Gray +#             Go to next file
-    #Gray -#             Go to previous file
-    #Ctrl+O#             Show user screen
-    #Ctrl+Alt+Shift#     Temporarily show user screen
-                       (as long as these keys are held down)
-    #Ctrl+B#             Show/Hide functional key bar at the bottom
-                       line.
-    #Ctrl+Shift+B#       Show/Hide status line
-    #Ctrl+S#             Show/Hide the scrollbar.
-    #Alt+BS, Ctrl+Z#     Undo position change
-    #RightCtrl+0..9#     Set a bookmark 0..9 at the current position
-    #Ctrl+Shift+0..9#    Set a bookmark 0..9 at the current position
-    #LeftCtrl+0..9#      Go to bookmark 0..9
-
-    #Ctrl+Ins, Ctrl+C#   Copy the text highlighted as a result of
-                       the search or manual selection to the clipboard.
-    #Ctrl+U#             Remove the highlighting of the selected text.
-    #Shift+Mouse click#  Manual text selection.
-                       If the mouse cursor is over the text, the first
-                       click marks the start of the selection and the
-                       second click marks the end of the selection. Use
-                       navigation keys after the first click to bring
-                       the end position into the view. The end position
-                       may be set earlier than the start.
-
-    See also the list of ~macro keys~@KeyMacroViewerList@, available in the viewer.
+  Navigation keys
+
+  The behavior of navigation keys depends on the ~view mode~@ViewerMode@.
+
+  The following keys work in all modes:
+
+    #Up#                 ^<wrap>Line up
+    #Down#               ^<wrap>Line down
+    #PgUp#               ^<wrap>Screenful up
+    #PgDn#               ^<wrap>Screenful down
+    #Home, Ctrl+Home#    ^<wrap>Beginning of the file
+    #End, Ctrl+End#      ^<wrap>End of the file
```

Comment:
Достаточно указать маркер `CtrlStartPosChar` (`^<wrap>`) в первой строке блока: https://api.farmanager.com/ru/language/control_statements.html

Reply:
I suggest using the following style:
```
 #Key1#, #Key2#
 Description description description  description  description
description  description  description  description.
```
(also see @SystemSettings topic).
It is flexible without littering the text with wrap markers and I'm about to reformat the whole hlf that way.

Reply:
@johnd0e, Thank you for the pointer (though the feature is not obvious from the spec). I removed the clutter.

@alabuzhev, There are 32 commands listed in the `Viewer commands` block alone. With the formatting you suggested it'd take at least 96 lines of the formatted help text; about two screens of normal height. I am really not sure if it would be a good use of the screen real estate. I can think of splitting the topic into four or five pages linked from the main page, but then it'll be too many clicks to read the entire topic. Please let me know. I'll be happy to reformat. Personally, I think that this iteration (01571856) is optimal, though not perfect.

Reply:
> the feature is not obvious from the spec)

Right, the exact behavior is not specified.
And I'd prefer if action of single marker continued down till the end of topic.

Reply:
Re "a good use of the screen real estate" - it's not a presentation and we don't need to fit everything on a slide. It should be readable and maintainable.
Two column layout looks ridiculous on small screen sizes for basically anything less trivial than `#F1#     Help` because the 1st column is often too wide due to one or two long strings and you can't fit more than 4-5 words into the 2nd one:
![image](https://user-images.githubusercontent.com/11453922/53091068-e2a53880-3508-11e9-9998-70c5ebd06489.png)


Note that `NumPad5,F3,F10,Esc` isn't even correctly punctuated - it should be `NumPad5, F3, F10, Esc`, but that would reduce the width of the 2nd column even more.

Also, the larger the gap, the harder it is to tell which key corresponds to which action (see `Alt+F11` vs `Gray+`). You have to move your eyes left and right several times till you get it.

And, finally, reformatting the whole topic every time we add a new key and there's no room for it (e.g. NumPad5,F3,F10,Esc,**Ctrl+Shift+Space**) is a PITA and kills any motivation to update the help.

(I'm not asking to change this now - it's more like an advance notice that it will be changed anyway one day)

Reply:
I agree with all your points... except for readability. When I showed my wife both variants side by side, she immediately and with absolute certainty said that columnar formatting is better.

Anyway, I created the parallel PR, #89, with the same contents but with the flow-style formatting. Please decide which one to merge (after fixing whatever comments may be).

Reply:
Ok, let's trust hallway testing and proceed with this one :)